XMOJ.user.js

```diff
@@ -1009,7 +1008,7 @@ if (document.querySelector("#navbar") != null) {
                     }
                     let OpenUnsolvedButton = document.createElement("button");
                     OpenUnsolvedButton.className = "btn btn-outline-secondary";
-                    OpenUnsolvedButton.innerText = "打开未解决题目";
+                    OpenUnsolvedButton.innerText = "打开未解决题目（OI模式下请手动打开题目）";
```

Comment:
I think you misunderstood the meaning of "Unsolved problem". Unsolved problem including problems that not judged or ac and problems that not submitted.